[Merged by Bors] - Enable configuration and environment overrides (#173)
stackabletech/superset-operator: docs/modules/ROOT/pages/usage.adoc

The `spec.loadExamplesOnInit` key is optional and defaults to `false`, it can be set to `true` to load example data into superset when the database is initialized.

> The `rowLimit` configuration option defines the row limit when requesting chart data.

Reviewer: ```suggestion
The `rowLimit` configuration option defines the row limit when reading data from a data source.
```

Author: see comment above
